### PR TITLE
Add server load utilities

### LINKED ISSUE #20: Add server load metrics
This will include a class to fetch the data from the system, a metrics gauge for storing the data and a task for running the check

### PR DESCRIPTION
**Pull request by chrisrohr**
This includes:
* fetcher to call out to uptime
* gauge to track load averages in metrics
* server task to expose an endpoint to retrieve current average

Closes #20

### CODE REVIEW

File: src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+     * Expected format:
+     * <p>
+     * {@code 18:29:28 up 34 days, 18:25, 1 user, load average: 0.88, 0.98, 1.03}
+     *
+     * @return an {@link Optional} with the load average string or {@link Optional#empty()} if not found
+     */
+    public Optional<String> get() {
+        var process = processes.launch("uptime");
+
+        try (var reader = new BufferedReader(new InputStreamReader(process.getInputStream(), UTF_8))) {
```
Comment (sleberknight) [inline src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java:53]: We should change these to use the utilities in `KiwiIO`.

File: src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        this(new ProcessHelper());
+    }
+
+    @VisibleForTesting
+    ServerLoadFetcher(ProcessHelper processes) {
+        this.processes = processes;
+    }
+
+    /**
+     * Expected format:
```
Comment (sleberknight) [inline src/main/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcher.java:44]:
```suggestion
     * Example format on Linux-based systems:
```

File: src/test/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcherTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    void testGet_UsingActualProcess() {
+        String load = new ServerLoadFetcher().get().orElse(null);
+        assertThat(load)
+                .isNotNull()
+                .matches("^\\d+.\\d+[,]? \\d+.\\d+[,]? \\d+.\\d+$");
+    }
+
+    @Test
+    void testGet_UsingMockProcess() throws InterruptedException {
+        String output = " 18:29:28 up 34 days, 18:25, 1 user, load average: 0.88, 0.98, 1.03";
```
Comment (sleberknight) [inline src/test/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcherTest.java:38]: You can `var` this...is there supposed to be a leading space?

Comment (chrisrohr) [reply src/test/java/org/kiwiproject/dropwizard/util/metrics/ServerLoadFetcherTest.java:38]: Ok, I don't think it matters, but I can remove it

**Review by sleberknight — APPROVED** [review]

**Review by sleberknight — APPROVED** [review]

**Pull request merged**